### ISSUE TITLE
[Accessibility] Selecting days using Voice Over does not work for custom DayViews

### ISSUE THREAD

**Issue by Blackjacx** [airbnb/HorizonCalendar#189]
I already testsed this with version `1.14.2` which is the newest at the moment.

When I have a calendar with custom content for:
- month header
- week of day header
- days

the selection of days does not work anymore when I swipe through the views using Voice Over. As you can see we have days that are not selectable in the screenshot below. The first problem is:
- as soon as I set e.g. `invariantViewProperties.highlightShapeDrawingConfig.fillColor = .red` the days become unselectable (not sure what the one has to do with the other)
- when I comment this line days become selectable but also the inactive ones. It would be cool when inactive days are skipped automatically by VO.

![5C78B0DA-4466-4689-B3E9-5EC3BE1D2DA4](https://user-images.githubusercontent.com/794372/204301485-f764c5cf-b366-40b4-9489-c120003f208e.jpeg)
